Types/ModuleInstallDescriptor.cs

```diff
@@ -186,14 +198,15 @@ public ModuleInstallDescriptor ConvertFindToFile(ZipFile zipfile)
             if (candidates.Count == 0)
             {
                 throw new FileNotFoundKraken(
-                    this.find,
-                    String.Format("Could not find {0} directory in zipfile to install", this.find)
+                    this.find ?? this.find_regexp,
```

Comment:
@RichardLake I think you may be referencing `?.`, the null-conditional operator. `??` is the null-coalesce operator and has been around since the beginning.